sdk/hardwaresecuritymodules/Azure.ResourceManager.HardwareSecurityModules/README.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure Hardware Security Modules management client library for .NET
+
+**[Describe the service briefly first.]**
```

Comment:
Please provide the description of your service here

Reply:
Description added